src/Mapping/ExtensionMetadataFactory.php

```diff
@@ -201,4 +194,18 @@ protected function getDriver($omDriver)
 
         return $driver;
     }
+
+    private function storeConfiguration(string $className, array $config): void
+    {
+        // Cache the result, even if it's empty, to prevent re-parsing non-existent annotations.
+        $cacheId = self::getCacheId($className, $this->extensionNamespace);
+
+        if (null === $this->cachePoolItem) {
```

Comment:
Move this up?  No point in computing a cache ID if the cache pool isn't set.

Reply:
thanks! in a previous version I was setting the configuration to the metadata cache when `$this->cachePoolItem` was null I forgot to move it again.